announcement/announcement-tool/tool/src/java/org/sakaiproject/announcement/tool/AnnouncementActionState.java

```diff
@@ -343,8 +343,22 @@ else if (VarNameDisplaySelection.equals("sortableWithBody"))
 					setShowAllColumns(true);
 					setShowAnnouncementBody(true);
 
-					setNumberOfCharsPerAnnouncement(numberOfCharsPerAnnouncement);
-					setEnforceNumberOfCharsPerAnnouncement(false);
+					String varNameNumberChars = parameters.getString("changeChars");
+					if (varNameNumberChars.equals(rb.getString("custom.shofir")))
```

Comment:
Fix identation please.

if () {

Reply:
I think this is fine here, the rest of this file uses this style. However I do agree that it might be better as a `switch`